packages/office-ui-fabric-react/src/components/Popup/Popup.types.ts

```diff
@@ -50,4 +50,11 @@ export interface IPopupProps extends React.HTMLAttributes<Popup> {
    * @default true
    */
   shouldRestoreFocus?: boolean;
+
+  /**
+   * If set to true, the popup will explicitly check if a scroll bar is needed and set overflow-y to scroll if necessary. This is useful
+   * when you don't want the scroll bar to take away space from the popout content, which is what happens when you use overflow-y:auto
+   * on some browsers.
+   */
+  shouldSetOverflowY?: boolean;
```

Comment:
you're not actually using this param anywhere in code

Reply:
hmm yes can we remove this?

Reply:
Remove or make it read It? I meant to make this optional behavior, so I was going to make the prop work